### PR TITLE
app-cdr/bchunk: EAPI="6", remove boilerplate, add ~x64-macos

### PR DESCRIPTION
**Pull request by yegortimoshenko**
/cc @mgorny 

I also would like to take maintainership over this package.

### CODE REVIEW

File: app-cdr/bchunk/bchunk-1.2.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+EAPI="6"
+
+DESCRIPTION="Convert CD images from bin/cue to iso+wav/cdr"
+HOMEPAGE="http://he.fi/bchunk/"
+SRC_URI="${HOMEPAGE}${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="amd64 ppc sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-macos ~sparc-solaris ~x86-solaris"
+
```
Comment (monsieurp) [inline app-cdr/bchunk/bchunk-1.2.0-r2.ebuild]: define a `DOCS=( ... )` array here and stuff it with everything you've given to `dodoc`.

File: app-cdr/bchunk/bchunk-1.2.0-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+src_install() {
+	dobin bchunk
+	doman bchunk.1
+	dodoc ${P}.lsm ${PN}.spec README ChangeLog
```
Comment (monsieurp) [inline app-cdr/bchunk/bchunk-1.2.0-r2.ebuild]: remove this line and call `einstalldocs` instead.

File: app-cdr/bchunk/bchunk-1.2.0-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+DESCRIPTION="Convert CD images from bin/cue to iso+wav/cdr"
+HOMEPAGE="http://he.fi/bchunk/"
+SRC_URI="${HOMEPAGE}${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="amd64 ppc sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos ~x64-macos ~sparc-solaris ~x86-solaris"
+
+src_install() {
+	dobin bchunk
```
Comment (monsieurp) [inline app-cdr/bchunk/bchunk-1.2.0-r2.ebuild]: bchunk -> `${PN}`, same thing below.

Comment (mgorny) [inline app-cdr/bchunk/bchunk-1.2.0-r2.ebuild]: Killing readability in the name of cool variable substitutions is not a valuable suggestion.

**Pull request closed without merging**